### PR TITLE
Added accessor methods for layer op attributes

### PR DESCRIPTION
**Pull request by jbobba**

### CODE REVIEW

File: src/ngraph/op/experimental/layers/reorg_yolo.hpp [hunk 1]
```diff
@@ -36,6 +36,7 @@ namespace ngraph
             virtual std::shared_ptr<Node>
                 copy_with_new_args(const NodeVector& new_args) const override;
 
+            const Strides get_stride() const { return m_stride; }
```
Comment (aprocter) [inline src/ngraph/op/experimental/layers/reorg_yolo.hpp]: Elsewhere in nGraph we've tended to say `get_strides()`.

File: src/ngraph/op/experimental/layers/psroi_pooling.hpp [hunk 2]
```diff
@@ -47,6 +47,11 @@ namespace ngraph
             virtual std::shared_ptr<Node>
                 copy_with_new_args(const NodeVector& new_args) const override;
 
+            size_t get_output_dim() const { return m_output_dim; }
```
Comment (aprocter) [inline src/ngraph/op/experimental/layers/psroi_pooling.hpp]: Any deep reason for not using a `struct` on this one?

File: src/ngraph/op/experimental/layers/roi_pooling.hpp [hunk 3]
```diff
@@ -43,6 +43,9 @@ namespace ngraph
             virtual std::shared_ptr<Node>
                 copy_with_new_args(const NodeVector& new_args) const override;
 
+            const Shape& get_output_size() const { return m_output_size; }
+            const float get_spatial_scale() const { return m_spatial_scale; }
```
Comment (aprocter) [inline src/ngraph/op/experimental/layers/roi_pooling.hpp]:
```suggestion
            float get_spatial_scale() const { return m_spatial_scale; }
```

File: src/ngraph/op/experimental/layers/psroi_pooling.hpp (hunk 2 above)
Comment (jbobba) [reply src/ngraph/op/experimental/layers/psroi_pooling.hpp]: Not really. In general, if the number of attributes crosses a certain pain threshold, I packed them in a struct.

**Pull request merged**